### PR TITLE
Allow only downscaling for past challenge workers

### PR DESCRIPTION
**Pull request by gchhablani**
This PR modifies the auto-scaling script in order to only allow downscaling for the past challenge workers.

### CODE REVIEW

File: scripts/monitoring/auto_scale_workers.py [hunk 1]
```diff
@@ -106,37 +141,15 @@ def increase_or_decrease_workers(challenge):
...
-                    challenge["id"], challenge["title"]
-                )
-            )
 
+    if (
+        queue_length == 0
+        or datetime.fromisoformat(challenge["end_date"][:-1])
+        < datetime.utcnow()
+    ):
+        decrease_workers(challenge, num_workers)
```
Comment (Ram81) [inline scripts/monitoring/auto_scale_workers.py:150]: can you rename the method to `scale_down_workers`?

File: scripts/monitoring/auto_scale_workers.py [hunk 2]
```diff
@@ -106,37 +141,15 @@ def increase_or_decrease_workers(challenge):
...
-                    challenge["id"], challenge["title"]
-                )
-            )
-        else:
-            print(
-                "Existing workers and pending queue messages found for Challenge ID: {}, Title: {}. Skipping.".format(
-                    challenge["id"], challenge["title"]
-                )
-            )
+        increase_workers(challenge, num_workers)
```
Comment (Ram81) [inline scripts/monitoring/auto_scale_workers.py:152]: can you rename the method to `scale_up_workers`?

File: scripts/monitoring/auto_scale_workers.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+from datetime import datetime
```
Comment (Ram81) [inline scripts/monitoring/auto_scale_workers.py:1]: Can we have all the imports starting with `from` after the `import <package>` statements?

**Review by Ram81 — CHANGES_REQUESTED** [review]
rest of the PR looks good. Minor comments

**Comment by gchhablani** [comment]
@Ram81 I have changed `increase_or_decrease` to `scale_up_or_down` as well. Hope that's okay.

**Review by Ram81 — APPROVED** [review]

**Pull request merged**